### PR TITLE
Don't open any PR's right now [Freeze]

### PR DESCRIPTION
**Pull request by optimumtact**
There's a complete freeze so for now please don't bother opening new PR's

change unrelated.

### CODE REVIEW

File: code/game/sound.dm [hunk 1]
```diff
@@ -23,6 +23,57 @@
 	)
 	environment = SOUND_ENVIRONMENT_NONE //Default to none so sounds without overrides dont get reverb
 
+/**
+ * Contains data required to update spatial audio awareness.
```
Comment (LemonInTheDark) [inline code/game/sound.dm:27]: I have BEEN saying this

**Comment by MrDoomBringer** [comment]
what for

**Comment by SyncIt21** [comment]
Not even fix PR's?

**Comment by Exester509** [comment]
> what for

#85491

**Comment by MrStonedOne** [comment]
> > what for
> 
> #85491

I know you think you are helping, but this doesn't answer the question.

*WHY* does the wallening require not even opening new prs?

**Comment by SyncIt21** [comment]
> good question
> I hope this get clarified soon enough by orange or someone

I think it's because of the large number of open PR's [171 as of this point] that are not getting merged because maintainers are putting all their focus on wallening.

Maybe when they start getting merged can we make space for new one's cause it's putting a lot of strain on reviewers

**Comment by necromanceranne** [comment]
I thought it was because of conflicts potentially happening with the wallening branch, and so prs aren't getting merged and too many prs means a lot of work post wallening to get everything reviewed and merged. We had a few prs slip in but they weren't conflicting so it was whatever. But wallening touches enough things that conflicts would get really annoying.

**Comment by btcbuster** [comment]
Would you mind telling us WHY you are doing this?

**Comment by zoomachina** [comment]
> Would you mind telling us WHY you are doing this?

image

**Comment by btcbuster** [comment]
On a serious note, its probably a good habit to provide reasoning for any sort of changes or policy on the github

**Comment by optimumtact** [comment]
> _WHY_ does the wallening require not even opening new prs?

Because people keep dropping into coderbus to ask why their PR is not merged so it's easier to just communicate to them once.

Also, we shouldn't let people have any expectation of a review or merge while they're not getting one.

**Comment by LemonInTheDark** [comment]
Wallening is merged and thus this serves no purpose unless we want to do something different with it

**Pull request closed without merging**